kolibri_explore_plugin/assets/src/components/DiscoveryNavBar.vue

```diff
@@ -2,23 +2,30 @@
 
   <NavBar class="discovery-navbar">
     <img class="logo mr-3" :src="logo">
-    <b-button-group>
-      <b-nav-text class="btn pl-0 text-primary" @click="goBack">
+    <b-button-group class="mx-auto">
```

Comment:
This is not centering the content in the whole bar space, it's centering in the available space, so it's not really centered because the right widget is larger, so it's placed a bit to the left.

Maybe we can use this simple css positioning that I've found in [stackoverflow](https://stackoverflow.com/a/42181069):

```
.abs-center-x {
    position: absolute;
    left: 50%;
    transform: translateX(-50%);
}
```

Reply:
Thanks for checking! I really dislike the absolute positioning for this. Let's ask Joana when she's back and polish accordingly.